edx_proctoring/templates/proctored_exam/onboarding_error.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam {{onboarding_status}}" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You must complete an onboarding exam before taking this proctored exam
+    {% endblocktrans %}
+  </h3>
+  <p>
+{% if onboarding_status == 'onboarding_missing' or onboarding_status ==  'onboarding_expired' %}
+    {% blocktrans %}
+    Please complete the <a href="{{onboarding_link}}">Onboarding Exam</a> before attempting this exam.
+    {% endblocktrans %}
+{% elif onboarding_status == 'onboarding_pending' %}
+    {% blocktrans %}
+    Your onboarding exam is being reviewed. Please allow 1-2 days before attempting this exam.
```

Comment:
Nit: This is, again, a copy question, but I'd tack something onto the end of "Please allow 1-2 days" to be a little more descriptive. Maybe "Please allow 1-2 days for onboarding exam review" or something. Just wanted to flag that.